core/src/apps/cardano/helpers/utils.py

```diff
@@ -31,4 +35,7 @@ def to_account_path(path: List[int]) -> List[int]:
 
 
 def to_account_number(path: List[int]) -> int:
+    if not SCHEMA_ACCOUNT_LOOSE.match(path):
+        raise ValueError
```

Comment:
I would add a message e.g. "Invalid path" unless message-less ValueErrors are a pattern used here

Reply:
Yes, I tried searching for similar errors, but it didn't lead to any conclusion. I also prefer to have the message there though, so added in https://github.com/vacuumlabs/trezor-firmware/pull/45/commits/47e255e808bc814d3bfd5b374cdfa9ceffab780d.